src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -37,12 +37,12 @@
           sm="7"
         >
           <v-text-field
-            class="identifier-textfield"
+            class="incorporation-number"
             filled
             hide-details
             label="B.C. Extraprovincial Incorporation Number"
             readonly
-            :value="identifier"
+            :value="incorporationNumber"
```

Comment:
This is this component:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/235c3482-d823-4fa9-9b02-afbe9052d873)